app/user_manager.py

```diff
@@ -81,7 +83,7 @@ def add_user(self, name):
         self.users[user_id] = name
 
         global users_file
```

Comment:
this can be removed, probably should check for other stray globals in this file

Reply:
Good catch. Removed it and other references in this file with the getter function `get_users_file`.